src/module.c

```diff
@@ -1734,7 +1777,7 @@ void moduleReplicateMultiIfNeeded(RedisModuleCtx *ctx) {
  * the AOF or the replicas from the propagation of the specified command.
  * Otherwise, by default, the command will be propagated in both channels.
  *
- * ## Note about calling this function from a thread safe context:
+ * #### Note about calling this function from a thread safe context:
```

Comment:
??

Reply:
Since the section headings are ## and the RM function name is ###, and this is a sub-headings inside the RM function documentation....